app/src/main/java/org/fossasia/openevent/general/auth/AuthService.kt

```diff
@@ -76,6 +76,14 @@ class AuthService(
                 }
     }
 
+    fun updateProfile(id: Long = authHolder.getId()): Single<User> {
```

Comment:
Where is it updating the profile?

Reply:
Inserting in the database and updating in the profile fragment without showing progress.

Reply:
That's fetching/saving/syncing

Updating is if you sent come data on server to update/

Reply:
Not getting it. Mean retrieving information from the server and updating the same on the server.

Reply:
> come data

Data coming from API?

Reply:
Updated

Reply:
But here we are updating User name only in edit profile fragment which is already updating on server. And now syncing the data is required because profile verification is working from the server side.

Reply:
Why is the name of method updateProfile?

Reply:
Oh got it. It shouldn't. It can be sync or fetch profile.

Reply:
Updated